build.wake

```diff
@@ -15,57 +15,87 @@ def macrosParadiseIvyDep =
   readIvyDepsJSON here
   | rmapPass extractIvyDep
 
-# TODO Should this be in the standard lib?
-def orElse b = match _
-  Some a = Some a
-  None   = b
-
 # Adds an a compiler plugin dependency on macros paradise
 # If module is already broken, keeps old error
 global def addMacrosParadiseCompilerPlugin module =
   match macrosParadiseIvyDep
     Pass dep = module | editScalaModuleCompilerPlugins (dep, _)
     Fail err = module | editScalaModuleError (orElse (Some err))
 
+# TODO This should be default behavior in api-scala-sifive
+def scalaModuleAddExtraSourceDirs module =
```

Comment:
I think it might be clearer to rename this to something that it indicates that it adds a scala-version-specific path to the source file search. `scalaModuleAddExtraSourceDirs` otherwise sounds like a function that would take another argument for the list of source dirs to add. Maybe `scalaModuleAddScalaVersionSourceDir`? If this is an actual named pattern in the Scala ecosystem, then it might be good to adopt that name.

Reply:
Renamed to `scalaModuleAddScalaVersionSourceDirs` (I left it plural because while there's only 1 right now, there are more that I could have it add and I don't want to rename it if I need to do that).